students/diego-guzman/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <a name="top"></a>
+    <a name="second"></a>
+    <h1>Welcome to the multiverse</h1>
+   <img src="https://www.poppastring.com/blog/content/binary/Windows-Live-Writer/ea115c478e3d_12972/Interstellar-Tesseract_thumb_1.jpg" alt="dimensions"/>
+    <!--<img src="1.png" alt="dimensions"/>-->
+    <h2>"Accident is the first building block of evolution."</h2>
+
+    <p><u>A machine</u> doesn't improvise well because <b>you cannot program a fear</b> of <i>death</i>. Our survival instinct is our greatest source of <strong>inspiration</strong></p>
+
+    <h2>Useless links</h2>
+<div>
+    <ul> 
+        <li><a href="mailto:blablabla@gmail.com"> Link to email</a> </li>
+        <li><a href="https://google.com">  Link to google</a> </li> 
+        <li><a href="tel:+341122333444"> Link to phone</a>  
+</div>
+
+<div>
+<label for="text">What's your name?</label>           
```

Comment:
`for`attribute value must match with the `id` atrtibute of the input field it belongs